app/src/main/java/com/duckduckgo/app/browser/omnibar/TopAppBarBehavior.kt

```diff
@@ -41,8 +44,11 @@ class TopAppBarBehavior(
         )
     }
 
+    @SuppressLint("RestrictedApi")
     override fun layoutDependsOn(parent: CoordinatorLayout, child: AppBarLayout, dependency: View): Boolean {
-        if (!viewsExemptedFromOffset.contains(dependency.id)) {
+        if (dependency is Snackbar.SnackbarLayout) {
+            updateSnackbar(child, dependency)
```

Comment:
As we discussed, this approach goes effectively against how coordinator layout should operate. At this point we could just add a layout observer.

That said, we're already using this same approach for the bottom bar, and we're hoping to get rid of some of the coordinator layout complexity soon, so let's just keep moving.